src/serac/physics/utilities/functional/tests/functional_comparison_L2.cpp

```diff
@@ -30,8 +30,11 @@ std::unique_ptr<mfem::ParMesh> mesh3D;
 template <int p, int dim>
 void functional_test(mfem::ParMesh& mesh, L2<p> test, L2<p> trial, Dimension<dim>)
 {
-  [[maybe_unused]] static constexpr double a = 1.7;
-  [[maybe_unused]] static constexpr double b = 0.0;
+  [[maybe_unused]] static constexpr double a       = 1.7;
+  [[maybe_unused]] static constexpr double b       = 0.0;
+  std::string                              postfix = serac::profiling::concat("_L2<", p, ">");
```

Comment:
This could be more consistent about keeping or omitting the `serac::profiling::` prefix, given the `using namespace serac::profiling;`

Later on, `concat` appears without explicitly specifying its namespace.